superset-frontend/src/chart/Chart.jsx

```diff
@@ -212,6 +215,7 @@ class Chart extends React.PureComponent {
           <div
             className={`slice_container ${isFaded ? ' faded' : ''}`}
             data-test="slice-container"
+            style={{ width, overflow: 'scroll' }}
```

Comment:
```suggestion
            style={{ width, overflow: 'auto' }}
```
does auto work just as well here, so we don't _always_ have a scrollbar?

Reply:
many visualization plugins may end up overflowing by a few pixels for whatever reason and we may not want to show the scrollbar.